lib/susedistribution.pm

```diff
@@ -18,11 +18,17 @@ use utils qw(
 use version_utils qw(is_hyperv_in_gui is_sle is_leap is_svirt_except_s390x is_tumbleweed is_opensuse);
 use x11utils qw(desktop_runner_hotkey ensure_unlocked_desktop x11_start_program_xterm);
 use Utils::Backends;
-use backend::svirt qw(SERIAL_TERMINAL_DEFAULT_DEVICE SERIAL_TERMINAL_DEFAULT_PORT);
+
 use Cwd;
 use autotest 'query_isotovideo';
 use isotovideo;
 
+use backend::svirt qw(SERIAL_TERMINAL_DEFAULT_DEVICE SERIAL_TERMINAL_DEFAULT_PORT);
+
+if (isotovideo::get_version() >= 39) {
```

Comment:
And here also CI complains about `Use of uninitialized value in numeric ge (>=)` :(.

Reply:
I would not use `if (isotovideo::get_version() >= 39)`, just define it as everything else, we can wait until workers are updated, it's not like we can have this only on some workers.

Reply:
Other option would be to detect if `eval "use use backend::svirt qw(SERIAL_USER_TERMINAL_DEFAULT_DEVICE SERIAL_USER_TERMINAL_DEFAULT_PORT)";` works. But ok, if it's not a problem, I'll just define new variables and we will wait till workers are updated. Thanks!

Reply:
Ye, but IMO we don't need to complicate it.